tests/test_funcs.py

```diff
@@ -23,7 +25,7 @@
 from array_api_extra._lib._utils._typing import Array, Device
 
 # some xp backends are untyped
-# mypy: disable-error-code=no-untyped-usage
+# mypy: disable-error-code=no-untyped-def
```

Comment:
This is to fix vscode's mypy plugin, which otherwise highlights the whole module in red.